Makefile

```diff
@@ -32,14 +38,16 @@ push-to-kind:
 
 .PHONY: chart-push
 chart-push:
-	helm chart save ./helm/afm ${CHART_IMG}
-	helm chart push ${CHART_IMG}
-	helm chart remove ${CHART_IMG}
+	helm package ${REPOSITORY} --destination=${TEMP} --version=${HELM_TAG}
+	helm push ${TEMP}/${CHART}-${HELM_TAG}.tgz ${HELM_CHART_PATH}
+	rm -rf ${TEMP}/${CHART}-${HELM_TAG}.tgz
+
 
 .PHONY: deploy
 deploy:
-	helm install --set image.repository=${HELM_REPOSITORY} --set image.tag=${TAG} afm ./helm/afm
+	helm install --set image.repository=${IMG_REPOSITORY} --set image.tag=${DOCKER_TAG} afm ./helm/afm
```

Comment:
Do we need here the `IMG_REPOSITORY` and `DOCKER_TAG` which is something like `ghcr.io/fybrik/arrow-flight-module:latest`. Or, something like `ghcr.io/fybrik/arrow-flight-module-chart:${HELM_TAG}`?

Reply:
@Mohammad-nassar10 the `image.` values are for the docker image repository and tag. This is why `DOCKER_TAG` and `IMG_REPOSITORY` are used.